content/get-started/new-leads/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Set up your own community
+---
+
+There are many communities stablished within Galaxy based on [**region**](/community/#regional-communities) or [**domain of science **](/community/#regional-communities) but if you feel like you just don't see your own or don't feel identified with them you can create your own community!
+
+Below you'll find a suggested path to follow before you can start your own community, a list of Do's and Dont's and also useful links you should probably check out.
+
+### Path 
+
+#### Learn the basics about Galaxy
+
+First of all, you should  start with our dedicated [Get Started](/get-started/) page to get a head-start of what Galaxy is about. You'll find all the necessary resources and you could even participate in the [Galaxy Mentorship Network](#) to get started with a Galaxy Mentor.
+
+#### Learn about Tool Development
+Galaxy as many tools already installed and ready to be used but when creating your community you'll probably need specific tools for your Galaxy analysis. Don't worry, see our dedicated page for [Tool Authors](/tools/) where you'll find everything you'll need and also reach out to the [Tool Working Group](/community/wg/) via Gitter.
+
+Don't forget to follow the [Best Practices](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices.html) for writing Galaxy Tools.
+
+#### Get involved
+- Introduce yourself and dive in our Working Groups.
+- Participate in our World Wide Galaxy Events
+- We strongly recommend you create your community chat. Reach out to our Lobby in [Gitter](https://gitter.im/galaxyproject/Lobby) and ask the admins to create your Community Chat.
+  - Setting up a mailing list is also another available option by sending an email to outreach@galaxyproject.org altough we strongly recommend to use a Community Chat.
+
+### Do's and Dont's 
+
+This curated table was made with the help of Community member's experience in hope to avoid obstacles in your path to create your Community.
+
+| __DO__  | __DON'T__   |
+|---|---|
+|Be patient! Take the time to learn the basics about Galaxy. | Hurry and dive in straight  to create without knowing a lot about Galaxy and it's workarounds. |
+| Ask if a tool for what you need already exists| Get hazy with all your tool's possible needs as there are many installed already that can be close to what you would like.|
+|Request your tool to be reviewed, avoid managint your repo alone!| Be a lone wolf. Oherwise the main servers may not want to install your tool, as they want to make sure it follows standars they have defined for developing tools. |
+|When you happen to need a tool make the tool > write the training > set up an event.  Repeat events and take advantage of Galaxy’s infrastructure (Smorgasbord,CoFests,GCC,etc) to improve your tool and receive feedback from the Community.|Set to create very specific tools and avoid feedback, try not to overwhelm yourself!|
```

Comment:
space missing after comma:

(Smorgasbord,CoFests,GCC,etc) --> (Smorgasbord, CoFests, GCC, etc)

Reply:
I am not sure I fully understand the meaning of:

, try not to overwhelm yourself!

in the context of DON'T